source/includes/_build.md

```diff
@@ -15,28 +21,45 @@ https://github.com/hyperledger/iroha /path/to/iroha
 
 ### How to run development environment
 
-``` shell
-/path/to/iroha/scripts/run-iroha-dev.sh
-```
+Run the script `run-iroha-dev.sh`, contained in the folder `scripts`: `sh .../iroha/scripts/run-iroha-dev.sh`
 
-You will be attached to interactive environment for development and testing with `iroha` folder mounted from host machine.
+After you execute this script, following things happen:
 
-Docker environment will be removed when you logout from the container.
+ 1. It checks if you don't have containers with Iroha already running. If yes — it will reattach you to interactive shell. Assuming if you don't have any, then:
+ 2. The script will download iroha-docker-develop, redis and postgres images. Iroha image contains all development dependencies, and is based on top of ubuntu:16.04.
+ 3. Three containers are created and launched.
+ 4. The user is attached to the interactive environment for development and testing with `iroha` folder mounted from the host machine.
 
-### Build `iroha` and run tests
+<aside class="notice">
+Docker environment will be removed when you detach from the container.
+</aside>
 
-Build:
-```
+
+### Build project and run tests
+
+> Build
+
+``` bash
 cmake -H. -Bbuild;
 cmake --build build -- -j$(nproc)
 ```
 
-`irohad` and `iroha-cli` binaries will be in `./build/bin` directory.
+<aside class="notice">
```

Comment:
May be this it will be better to show this paragraph like a note? It looks like not a really important part of manual.

Reply:
It looks like a note